R/bootstrap-model.R

```diff
@@ -173,26 +165,27 @@ setup_bootstrap_run <- function(
 
     if(is.null(data)){
       # Only include subjects that entered the original problem by default
-      can_be_joined <- can_be_nm_joined(orig_mod, .join_col = .join_col)
-      if(isTRUE(can_be_joined)){
-        starting_data <- nm_join(orig_mod, .join_col = .join_col) %>%
-          suppressMessages()
+      starting_data <- nm_data(orig_mod, filter = TRUE) %>% suppressMessages()
 
-        # select only columns from original data set
-        starting_data <- starting_data %>%
-          dplyr::select(attr(starting_data, "nm_join_origin")$data)
-
-        if ("DV.DATA" %in% names(starting_data)) {
-          starting_data <- dplyr::rename(starting_data, "DV" = "DV.DATA")
-        }
-      }else{
+      # NULL if IGNORE/ACCEPT expressions cant be turned into dplyr expressions
+      if(is.null(starting_data)){
```

Comment:
The purpose of this PR was to move away from `nm_join` as the default via parsing the data record. I dont think we want to start requiring a dataset since this was not required in the previous release. That could be a change down the line, but I would consider that out of scope for this and the parent PR.

Reply:
Ahh I misinterpreted that - im good with that change. I just implemented your suggestion in the latest commit.